pkg/cloud/gen.go

```diff
@@ -36,6 +36,15 @@ import (
 	ga "google.golang.org/api/compute/v1"
 )
 
+const (
+	// regionalKeyFmt is the key format of regional resources in the AggregateList response
+	regionalKeyFmt = "regions/%v"
```

Comment:
%s

Reply:
instead of separating out the fmt string, you should probably just do:
```
func aggregatedListKey(k meta.Key) string {
  switch ...
    case ...: return sprintf...
}
```